### PR TITLE
Set up dev tools

### PR DESCRIPTION
**Pull request by Caruki**
Set up dev tools: htmlhint, prettier and sass. 
Create index.html

image

### CODE REVIEW

**Review by florianGierlichs — CHANGES_REQUESTED** [review]
nvm

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
```
Comment (marwinburesch) [inline .gitignore:105]:
```suggestion
.prettierrc
```

**Review by marwinburesch — CHANGES_REQUESTED** [review]
lgtm, just one thing

File: .gitignore (hunk 1 above)
Comment (Caruki) [reply .gitignore:105]: .prettierrc is not supposed to be in gitignore... but thank you for the suggestion

**Review by Johannafendrich — APPROVED** [review]
looks good

**Pull request merged**